devopstools-jenkins-chefhabitat-kubernetes/azuredeploy.json

```diff
@@ -0,0 +1,417 @@
+{
+    "$schema": "http://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "adminUsername": {
+            "type": "string",
+            "metadata": {
+                "description": "user name for the virtual machines."
+            }
+        },
+        "azureUsername": {
+            "type": "string",
+            "metadata": {
+                "description": "azure login name"
+            }
+        },
+        "azurePassword": {
+            "type": "securestring",
+            "metadata": {
+                "description": "azure login password"
+            }
+        },
+        "authenticationType": {
+            "type": "string",
+            "defaultValue": "sshPublicKey",
+            "metadata": {
+                "description": "authenticationType for virtula machines"
+            }
+        },
+        "sshPublicKey": {
+            "type": "string",
+            "metadata": {
+                "description": "generate ssh public key"
+            }
+        },
+        "adminPassword": {
+            "type": "securestring",
+            "metadata": {
+                "description": "password for elk and vmss virtual machines"
+            }
+        },
+        "jenkinsVmSize": {
+            "type": "string",
+            "defaultValue": "Standard_DS2_v2",
+            "metadata": {
+                "description": "vm size"
```

Comment:
if we're requiring premium storage make sure to note that in the descriptions

Reply:
We are using Standard_LRS storage account for vm's.